src/components/elements/Checkbox/style.scss

```diff
@@ -0,0 +1,132 @@
+:local {
+  @import '../../../styles/config';
```

Comment:
@suwanny here is what I think. this components will be used in another app. So if we use absolute path for files, it starts from the app's root directory, not react-toolset's root directory. So, relative path should be used. What do you think?